CHANGELOG.md

```diff
@@ -176,6 +177,7 @@
 - PR #5656 Fix issue with incorrect docker image being used in local build script
 - PR #5671 Fix chunksize issue with `DataFrame.to_csv`
 - PR #5672 Fix crash in parquet writer while writing large string data
+- PR #5692 Fix compilation issue with gcc 7.4.0 and CUDA 10.1
```

Comment:
I imagine this isn't actually dependent, but just unblocks your development flow?

Reply:
Yep